citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AuthorityEntryController.java

```diff
@@ -71,8 +84,71 @@
             logger.info("Not a valid URI: " + uri, e);
             return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
         }
-     
-        entry = authorityService.create(entry, (IUser)authentication.getPrincipal());
+
+        entry = authorityService.create(entry, (IUser) authentication.getPrincipal());
+        return new ResponseEntity<IAuthorityEntry>(entry, HttpStatus.OK);
+    }
+    
+    @RequestMapping(value = "/auth/authority/conceptpower/create", method = RequestMethod.POST)
```

Comment:
once we have more than just conceptpower as an authority service, this method would probably also create authorities for other services than conceptpower, so the URL shouldn't hardcode conceptpower.